maxim/models/query_builder.py

```
@@ -75,14 +75,14 @@ def deployment_var(self, key: str, value: Union[str, int, bool, list], enforce:
         Args:
             key (str): The key of the deployment variable.
-            value (Union[str, int, bool]): The value of the deployment variable.
+            value (Union[str, int, bool, list]): The value of the deployment variable.
             enforce (bool, optional): Whether to enforce the deployment variable. Defaults to True.
 
         Returns:
             QueryBuilder: The current QueryBuilder instance for method chaining.
         """
 
-        if len(self.query) > 0:
+        if len(self.query)>0:
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)